bindings/web5_uniffi_wrapper/src/dids/methods/did_jwk.rs

```diff
@@ -9,18 +16,24 @@ pub fn did_jwk_resolve(uri: &str) -> Arc<ResolutionResult> {
     Arc::new(ResolutionResult(resolution_result))
 }
 
-impl DidJwk {
-    pub fn from_public_jwk(public_key: Jwk) -> Result<Self> {
-        let did_jwk = InnerDidJwk::from_public_jwk(public_key)?;
-        Ok(Self(did_jwk))
-    }
+#[derive(Default)]
+pub struct DidJwkCreateOptions {
+    pub key_manager: Option<Arc<dyn KeyManager>>,
+    pub dsa: Option<Dsa>,
+}
 
-    pub fn from_uri(uri: &str) -> Result<Self> {
-        let did_jwk = InnerDidJwk::from_uri(uri)?;
-        Ok(Self(did_jwk))
-    }
+pub fn did_jwk_create(options: Option<DidJwkCreateOptions>) -> Result<Arc<BearerDid>> {
+    let inner_options = match options {
```

Comment:
Since we're no-oping the `None` case, might be cleaner to use `.map()` instead of `match`

Reply:
done